custom_components/multiscrape/__init__.py

```diff
@@ -117,22 +106,25 @@ async def _async_process_config(hass: HomeAssistant, config) -> bool:
             file_manager = LoggingFileManager(folder)
             await hass.async_add_executor_job(file_manager.create_folders)
 
-        http = create_http_wrapper(config_name, conf, hass, file_manager)
-
         form_submit_config = conf.get(CONF_FORM_SUBMIT)
         form_submitter = None
         if form_submit_config:
-            form_http = create_http_wrapper(config_name, form_submit_config, hass, file_manager)
             parser = conf.get(CONF_PARSER)
+            form_http = create_http_wrapper(config_name, form_submit_config, hass, file_manager)
+            form_scraper = create_scraper(config_name, conf, hass, file_manager)
```

Comment:
Let's only create this one when variables are given in the config. Otherwise set it to None.

Reply:
Moved the logic to `create_form_submitter` and done that.